frontend/packages/core/src/quick-links.tsx

```diff
@@ -29,8 +29,14 @@ const StyledButton = styled("button")({
   display: "flex",
 });
 
-const StyledSpan = styled("span")({
-  whiteSpace: "nowrap",
+const StyledPopperItem = styled(PopperItem)({
+  height: "auto !important",
```

Comment:
Is this `!important` necessary?

Reply:
`PopperItem` has this high specificity selector, hence my `!important` and also since the parent is a `Popper` that doesn't really take in styles for its children.
So TIL in styled components we are able to nest `&` selectors and increase specificity that way.

Reply:
Oh interesting, I had no idea as well lol. But my CSS knowledge is limited anyways haha. 

Yeah I was thinking maybe it was similar to before where we had to make sure we allowed the className prop type like I had in a previous PR. Ultimately just want to avoid `!important` when possible :)